x-pack/plugins/rule_registry/server/utils/create_lifecycle_executor.ts

```diff
@@ -63,6 +64,10 @@ export type LifecycleAlertService<
   fields: ExplicitAlertFields;
 }) => Alert<InstanceState, InstanceContext, ActionGroupIds>;
 
+export interface AdditionalContext {
```

Comment:
nit: `AdditionalContext` doesn't make sense for the type name here since we're returning the entire alert document.

Reply:
true, have removed it.